### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by anup-nehe**
https://mosip.atlassian.net/browse/MOSIP-30230
added new testcases 
-valid context in the help page
-accessing help page in offline
-scrolling the help page
-deleteDownloadedVcInOfflineMode

### CODE REVIEW

**Review by lsivanand — APPROVED** [review]

File: injitest/src/main/java/io/mosip/test/mob/inji/pages/ConfirmPasscode.java [hunk 1]
```diff
@@ -11,10 +11,18 @@ public class ConfirmPasscode extends BasePage {
     @iOSXCUITFindBy(accessibility = "Confirm passcode")
     @AndroidFindBy(xpath = "//*[contains(@text,'Confirm passcode')]")
     private WebElement confirmPasscode;
+
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/io/mosip/test/mob/inji/pages/ConfirmPasscode.java:14]: remove extra space

File: injitest/src/main/java/io/mosip/test/mob/inji/pages/HelpPage.java [hunk 2]
```diff
@@ -26,5 +33,12 @@ public boolean isHelpPageLoaded() {
     public void exitHelpPage() {
         this.clickOnElement(crossIcon);
     }
-
+    
+    public void ScrollToViewActivityLog() {
+    	howToViewActivity.click();
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/io/mosip/test/mob/inji/pages/HelpPage.java:38]: use common method for click

File: injitest/src/main/java/io/mosip/test/mob/inji/pages/HelpPage.java [hunk 3]
```diff
@@ -26,5 +33,12 @@ public boolean isHelpPageLoaded() {
...
+    }
+    
+    public boolean isHowToAddCardLoad() {
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/io/mosip/test/mob/inji/pages/HelpPage.java:41]: change method name to isHelpPageEmpty()

File: injitest/src/main/java/io/mosip/test/mob/inji/pages/HelpPage.java [hunk 4]
```diff
@@ -14,7 +15,13 @@ public class HelpPage extends BasePage {
     @AndroidFindBy(xpath = "//*[@resource-id=\"iconIcon\"]")
     @iOSXCUITFindBy(iOSClassChain = "**/XCUIElementTypeOther[`label == \"\uE5CD\"`][2]")
     private WebElement crossIcon;
-
+    
+    @AndroidFindBy(uiAutomator = "new UiScrollable(new UiSelector()).scrollIntoView(text(\"How to view activity logs?\"));")
+    public WebElement howToViewActivity;
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/io/mosip/test/mob/inji/pages/HelpPage.java:20]: howToViewActivityHeading

Use gesture swipe or visibility check method

File: injitest/src/main/java/io/mosip/test/mob/inji/pages/HistoryPage.java [hunk 5]
```diff
@@ -27,11 +30,30 @@ private boolean verifyHistoryIos(String vcNumber) {
         By locator = By.xpath("//*[contains(@name,'" + vcNumber + " downloaded')]");
         return this.isElementDisplayed(locator, "Downloaded VC in ios");
     }
+    
+
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/io/mosip/test/mob/inji/pages/HistoryPage.java:34]: remve extra space

File: injitest/src/main/java/io/mosip/test/mob/inji/pages/HistoryPage.java [hunk 6]
```diff
@@ -27,11 +30,30 @@ private boolean verifyHistoryIos(String vcNumber) {
...
+    
 
     private boolean verifyHistoryAndroid(String vcNumber) {
         By locator = By.xpath("//*[contains(@text,'" + vcNumber + " downloaded')]");
         return this.isElementDisplayed(locator, "Downloaded VC in android");
     }
+    
+    private boolean verifyDeleteHistoryAndroid(String vcNumber) {
+        By locator = By.xpath("//*[contains(@text,'" + vcNumber + " Removed from wallet')]");
+        return this.isElementDisplayed(locator, "Downloaded VC in android");
```
Comment (kamalsinghthoughtworks) [inline injitest/src/main/java/io/mosip/test/mob/inji/pages/HistoryPage.java:43]: update log method for deleting VC

**Pull request closed without merging**